Sources/Misc/Codable/AnyEncodable.swift

```diff
@@ -69,3 +70,43 @@ extension AnyEncodable: Encodable {
     }
 
 }
+
+/// `AnyEncodable` can also be decoded
+extension AnyEncodable: Decodable {
```

Comment:
feels like we should make this AnyCodable now?

Reply:
I'm doing that in a separate PR cause it ended up a bit more work.

Reply:
#2770